# -*- coding: utf-8 -*-

import sys
#from PyQt5 import uic, QtWidgets
from PyQt5 import QtCore, QtGui, QtWidgets


sys.path.append('gui')

class Ui_CalibrationSetup_FS(object):
    #def __init__(self, parent=None):
        #super(Ui_Calibration_FS, self).__init__(parent)
        #self.setupUi()

        # define signals

    #def setupUi(self):
        #dialog = uic.loadUi("uifiles/versuchsaufbau_FS.ui", self)
        #dialog.show()

    def setupUi(self, TestSetUp):
        TestSetUp.setObjectName("TestSetUp")
        TestSetUp.setWindowModality(QtCore.Qt.WindowModal)
        TestSetUp.resize(1401, 792)
        TestSetUp.setMinimumSize(QtCore.QSize(1401, 792))
        TestSetUp.setMaximumSize(QtCore.QSize(1401, 792))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        TestSetUp.setFont(font)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("icon materials/4.png"), QtGui.QIcon.Selected, QtGui.QIcon.On)
        TestSetUp.setWindowIcon(icon)
        self.Warning = QtWidgets.QLabel(TestSetUp)
        self.Warning.setGeometry(QtCore.QRect(50, 720, 51, 51))
        self.Warning.setMinimumSize(QtCore.QSize(51, 51))
        self.Warning.setMaximumSize(QtCore.QSize(51, 51))
        self.Warning.setText("")
        self.Warning.setObjectName("Warning")
        self.TestSetUpGroup = QtWidgets.QTabWidget(TestSetUp)
        self.TestSetUpGroup.setGeometry(QtCore.QRect(20, 70, 1361, 631))
        self.TestSetUpGroup.setMinimumSize(QtCore.QSize(1361, 631))
        self.TestSetUpGroup.setMaximumSize(QtCore.QSize(1361, 631))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.TestSetUpGroup.setFont(font)
        self.TestSetUpGroup.setObjectName("TestSetUpGroup")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.label_Polar = QtWidgets.QLabel(self.tab)
        self.label_Polar.setGeometry(QtCore.QRect(30, 50, 150, 30))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.label_Polar.sizePolicy().hasHeightForWidth())
        self.label_Polar.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setFamily("Verdana")
        self.label_Polar.setFont(font)
        self.label_Polar.setObjectName("label_Polar")
        self.comboBox = QtWidgets.QComboBox(self.tab)
        self.comboBox.setGeometry(QtCore.QRect(150, 55, 100, 30))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.comboBox.setFont(font)
        self.comboBox.setObjectName("comboBox")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.checkBox_linearity = QtWidgets.QCheckBox(self.tab)
        self.checkBox_linearity.setGeometry(QtCore.QRect(310, 50, 211, 30))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.checkBox_linearity.sizePolicy().hasHeightForWidth())
        self.checkBox_linearity.setSizePolicy(sizePolicy)
        self.checkBox_linearity.setObjectName("checkBox_linearity")
        self.label_FieldToler = QtWidgets.QLabel(self.tab)
        self.label_FieldToler.setGeometry(QtCore.QRect(590, 50, 100, 30))
        self.label_FieldToler.setObjectName("label_FieldToler")
        self.lineEdit_toler1 = QtWidgets.QLineEdit(self.tab)
        self.lineEdit_toler1.setGeometry(QtCore.QRect(690, 55, 100, 25))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.lineEdit_toler1.setFont(font)
        self.lineEdit_toler1.setAlignment(QtCore.Qt.AlignRight | QtCore.Qt.AlignTrailing | QtCore.Qt.AlignVCenter)
        self.lineEdit_toler1.setObjectName("lineEdit_toler1")
        self.label_durch = QtWidgets.QLabel(self.tab)
        self.label_durch.setGeometry(QtCore.QRect(790, 55, 25, 25))
        self.label_durch.setAlignment(QtCore.Qt.AlignCenter)
        self.label_durch.setObjectName("label_durch")
        self.lineEdit_toler2 = QtWidgets.QLineEdit(self.tab)
        self.lineEdit_toler2.setGeometry(QtCore.QRect(815, 55, 100, 25))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.lineEdit_toler2.setFont(font)
        self.lineEdit_toler2.setAlignment(QtCore.Qt.AlignRight | QtCore.Qt.AlignTrailing | QtCore.Qt.AlignVCenter)
        self.lineEdit_toler2.setObjectName("lineEdit_toler2")
        self.toolButton_new = QtWidgets.QToolButton(self.tab)
        self.toolButton_new.setGeometry(QtCore.QRect(30, 150, 130, 30))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(12)
        self.toolButton_new.setFont(font)
        self.toolButton_new.setObjectName("toolButton_new")
        self.toolButton_copy = QtWidgets.QToolButton(self.tab)
        self.toolButton_copy.setGeometry(QtCore.QRect(200, 150, 130, 30))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(12)
        self.toolButton_copy.setFont(font)
        self.toolButton_copy.setObjectName("toolButton_copy")
        self.toolButton_edit = QtWidgets.QToolButton(self.tab)
        self.toolButton_edit.setGeometry(QtCore.QRect(370, 150, 130, 30))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(12)
        self.toolButton_edit.setFont(font)
        self.toolButton_edit.setObjectName("toolButton_edit")
        self.toolButton_delete = QtWidgets.QToolButton(self.tab)
        self.toolButton_delete.setGeometry(QtCore.QRect(540, 150, 130, 30))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(12)
        self.toolButton_delete.setFont(font)
        self.toolButton_delete.setObjectName("toolButton_delete")
        self.treeWidget = QtWidgets.QTreeWidget(self.tab)
        self.treeWidget.setGeometry(QtCore.QRect(20, 200, 1310, 450))
        self.treeWidget.setObjectName("treeWidget")
        self.treeWidget.headerItem().setTextAlignment(0, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(0, font)
        self.treeWidget.headerItem().setTextAlignment(1, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(1, font)
        self.treeWidget.headerItem().setTextAlignment(2, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(2, font)
        self.treeWidget.headerItem().setTextAlignment(3, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(3, font)
        self.treeWidget.headerItem().setTextAlignment(4, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(4, font)
        self.treeWidget.headerItem().setTextAlignment(5, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(5, font)
        self.treeWidget.headerItem().setTextAlignment(6, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(6, font)
        self.treeWidget.headerItem().setTextAlignment(7, QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.treeWidget.headerItem().setFont(7, font)
        item_0 = QtWidgets.QTreeWidgetItem(self.treeWidget)
        item_0 = QtWidgets.QTreeWidgetItem(self.treeWidget)
        self.treeWidget.header().setVisible(True)
        self.treeWidget.header().setCascadingSectionResizes(False)
        self.treeWidget.header().setDefaultSectionSize(130)
        self.treeWidget.header().setHighlightSections(False)
        self.treeWidget.header().setSortIndicatorShown(False)
        self.treeWidget.header().setStretchLastSection(False)
        self.TestSetUpGroup.addTab(self.tab, "")
        self.TestSetUpSignalRouting = QtWidgets.QWidget()
        self.TestSetUpSignalRouting.setObjectName("TestSetUpSignalRouting")
        self.SignalTabel = QtWidgets.QTableWidget(self.TestSetUpSignalRouting)
        self.SignalTabel.setGeometry(QtCore.QRect(25, 160, 1311, 451))
        self.SignalTabel.setMinimumSize(QtCore.QSize(1311, 451))
        self.SignalTabel.setMaximumSize(QtCore.QSize(1311, 451))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        self.SignalTabel.setFont(font)
        self.SignalTabel.setVerticalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOn)
        self.SignalTabel.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
        self.SignalTabel.setObjectName("SignalTabel")
        self.SignalTabel.setColumnCount(5)
        self.SignalTabel.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.SignalTabel.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.SignalTabel.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.SignalTabel.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.SignalTabel.setHorizontalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setPointSize(10)
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.SignalTabel.setHorizontalHeaderItem(4, item)
        self.NewButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.NewButton.setGeometry(QtCore.QRect(470, 100, 131, 41))
        self.NewButton.setMinimumSize(QtCore.QSize(131, 41))
        self.NewButton.setMaximumSize(QtCore.QSize(131, 41))
        self.NewButton.setObjectName("NewButton")
        self.CopyButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.CopyButton.setEnabled(False)
        self.CopyButton.setGeometry(QtCore.QRect(610, 100, 131, 41))
        self.CopyButton.setMinimumSize(QtCore.QSize(131, 41))
        self.CopyButton.setMaximumSize(QtCore.QSize(131, 41))
        self.CopyButton.setObjectName("CopyButton")
        self.EditButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.EditButton.setEnabled(False)
        self.EditButton.setGeometry(QtCore.QRect(750, 100, 131, 41))
        self.EditButton.setMinimumSize(QtCore.QSize(131, 41))
        self.EditButton.setMaximumSize(QtCore.QSize(131, 41))
        self.EditButton.setObjectName("EditButton")
        self.DeleteButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.DeleteButton.setEnabled(False)
        self.DeleteButton.setGeometry(QtCore.QRect(890, 100, 131, 41))
        self.DeleteButton.setMinimumSize(QtCore.QSize(131, 41))
        self.DeleteButton.setMaximumSize(QtCore.QSize(131, 41))
        self.DeleteButton.setObjectName("DeleteButton")
        self.ImportButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.ImportButton.setGeometry(QtCore.QRect(1030, 100, 131, 41))
        self.ImportButton.setMinimumSize(QtCore.QSize(131, 41))
        self.ImportButton.setMaximumSize(QtCore.QSize(131, 41))
        self.ImportButton.setObjectName("ImportButton")
        self.systemTestButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.systemTestButton.setEnabled(False)
        self.systemTestButton.setGeometry(QtCore.QRect(1180, 100, 151, 41))
        self.systemTestButton.setMinimumSize(QtCore.QSize(151, 41))
        self.systemTestButton.setMaximumSize(QtCore.QSize(151, 41))
        self.systemTestButton.setCheckable(False)
        self.systemTestButton.setChecked(False)
        self.systemTestButton.setObjectName("systemTestButton")
        self.ExpandButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.ExpandButton.setGeometry(QtCore.QRect(30, 100, 171, 41))
        self.ExpandButton.setMinimumSize(QtCore.QSize(171, 41))
        self.ExpandButton.setMaximumSize(QtCore.QSize(171, 41))
        font = QtGui.QFont()
        font.setUnderline(True)
        self.ExpandButton.setFont(font)
        self.ExpandButton.setObjectName("ExpandButton")
        self.CollposeButton = QtWidgets.QPushButton(self.TestSetUpSignalRouting)
        self.CollposeButton.setGeometry(QtCore.QRect(210, 100, 241, 41))
        self.CollposeButton.setMinimumSize(QtCore.QSize(241, 41))
        self.CollposeButton.setMaximumSize(QtCore.QSize(241, 41))
        font = QtGui.QFont()
        font.setUnderline(True)
        self.CollposeButton.setFont(font)
        self.CollposeButton.setObjectName("CollposeButton")
        self.SystemControllerLabel = QtWidgets.QLabel(self.TestSetUpSignalRouting)
        self.SystemControllerLabel.setGeometry(QtCore.QRect(30, 50, 250, 31))
        self.SystemControllerLabel.setMinimumSize(QtCore.QSize(250, 31))
        self.SystemControllerLabel.setMaximumSize(QtCore.QSize(250, 31))
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.SystemControllerLabel.setFont(font)
        self.SystemControllerLabel.setObjectName("SystemControllerLabel")
        self.KontrollerComBoBox = QtWidgets.QComboBox(self.TestSetUpSignalRouting)
        self.KontrollerComBoBox.setGeometry(QtCore.QRect(320, 50, 1011, 31))
        self.KontrollerComBoBox.setMinimumSize(QtCore.QSize(1011, 31))
        self.KontrollerComBoBox.setMaximumSize(QtCore.QSize(1011, 31))
        self.KontrollerComBoBox.setEditable(True)
        self.KontrollerComBoBox.setObjectName("KontrollerComBoBox")
        self.TestSetUpGroup.addTab(self.TestSetUpSignalRouting, "")
        self.WarningLabel = QtWidgets.QLabel(TestSetUp)
        self.WarningLabel.setGeometry(QtCore.QRect(130, 730, 251, 31))
        self.WarningLabel.setMinimumSize(QtCore.QSize(251, 31))
        self.WarningLabel.setMaximumSize(QtCore.QSize(251, 31))
        self.WarningLabel.setObjectName("WarningLabel")
        self.ApplyButton = QtWidgets.QPushButton(TestSetUp)
        self.ApplyButton.setGeometry(QtCore.QRect(1070, 720, 131, 41))
        self.ApplyButton.setMinimumSize(QtCore.QSize(131, 41))
        self.ApplyButton.setMaximumSize(QtCore.QSize(131, 41))
        self.ApplyButton.setDefault(True)
        self.ApplyButton.setObjectName("ApplyButton")
        self.CancelButton = QtWidgets.QPushButton(TestSetUp)
        self.CancelButton.setGeometry(QtCore.QRect(1220, 720, 131, 41))
        self.CancelButton.setMinimumSize(QtCore.QSize(131, 41))
        self.CancelButton.setMaximumSize(QtCore.QSize(131, 41))
        self.CancelButton.setObjectName("CancelButton")

        self.retranslateUi(TestSetUp)
        self.TestSetUpGroup.setCurrentIndex(1)
        self.CancelButton.clicked['bool'].connect(TestSetUp.close)
        QtCore.QMetaObject.connectSlotsByName(TestSetUp)

        # define the background color of the window
        pale = QtGui.QPalette()
        pale.setColor(QtGui.QPalette.Background, QtGui.QColor(248, 248, 248))
        TestSetUp.setPalette(pale)

    def retranslateUi(self, TestSetUp):
        _translate = QtCore.QCoreApplication.translate
        TestSetUp.setWindowTitle(_translate("TestSetUp", "Versuchsaufbau"))
        self.label_Polar.setText(_translate("TestSetUp", "Polarization:"))
        self.comboBox.setItemText(0, _translate("TestSetUp", "Horizental"))
        self.comboBox.setItemText(1, _translate("TestSetUp", "Vertikal"))
        self.checkBox_linearity.setText(_translate("TestSetUp", "Verstärker Linearität checken"))
        self.label_FieldToler.setText(_translate("TestSetUp", "Feld Toleranz:"))
        self.lineEdit_toler1.setText(_translate("TestSetUp", "+4V/m"))
        self.label_durch.setText(_translate("TestSetUp", "/"))
        self.lineEdit_toler2.setText(_translate("TestSetUp", "0V/m"))
        self.toolButton_new.setText(_translate("TestSetUp", "Neu"))
        self.toolButton_copy.setText(_translate("TestSetUp", "Kopieren"))
        self.toolButton_edit.setText(_translate("TestSetUp", "Bearbeiten"))
        self.toolButton_delete.setText(_translate("TestSetUp", "Löschen"))
        self.treeWidget.headerItem().setText(0, _translate("TestSetUp", "Freq (Hz)"))
        self.treeWidget.headerItem().setText(1, _translate("TestSetUp", "Freq Step"))
        self.treeWidget.headerItem().setText(2, _translate("TestSetUp", "Level"))
        self.treeWidget.headerItem().setText(3, _translate("TestSetUp", "Level Step"))
        self.treeWidget.headerItem().setText(4, _translate("TestSetUp", "Dwell"))
        self.treeWidget.headerItem().setText(5, _translate("TestSetUp", "CW"))
        self.treeWidget.headerItem().setText(6, _translate("TestSetUp", "Modulation 1"))
        self.treeWidget.headerItem().setText(7, _translate("TestSetUp", "Modulation 2"))
        __sortingEnabled = self.treeWidget.isSortingEnabled()
        self.treeWidget.setSortingEnabled(False)
        self.treeWidget.topLevelItem(0).setText(0, _translate("TestSetUp", "80M"))
        self.treeWidget.topLevelItem(0).setText(1, _translate("TestSetUp", "1%"))
        self.treeWidget.topLevelItem(0).setText(2, _translate("TestSetUp", "30"))
        self.treeWidget.topLevelItem(0).setText(3, _translate("TestSetUp", "None"))
        self.treeWidget.topLevelItem(0).setText(4, _translate("TestSetUp", "1s"))
        self.treeWidget.topLevelItem(0).setText(5, _translate("TestSetUp", "Off"))
        self.treeWidget.topLevelItem(0).setText(6, _translate("TestSetUp", "AM Sine(1kHz, 80%)"))
        self.treeWidget.topLevelItem(0).setText(7, _translate("TestSetUp", "Off"))
        self.treeWidget.topLevelItem(1).setText(0, _translate("TestSetUp", "1G"))
        self.treeWidget.topLevelItem(1).setText(1, _translate("TestSetUp", "1%"))
        self.treeWidget.topLevelItem(1).setText(2, _translate("TestSetUp", "30"))
        self.treeWidget.topLevelItem(1).setText(3, _translate("TestSetUp", "None"))
        self.treeWidget.topLevelItem(1).setText(4, _translate("TestSetUp", "1s"))
        self.treeWidget.topLevelItem(1).setText(5, _translate("TestSetUp", "Off"))
        self.treeWidget.topLevelItem(1).setText(6, _translate("TestSetUp", "Off"))
        self.treeWidget.topLevelItem(1).setText(7, _translate("TestSetUp", "Off"))
        self.treeWidget.setSortingEnabled(__sortingEnabled)
        self.TestSetUpGroup.setTabText(self.TestSetUpGroup.indexOf(self.tab),
                                       _translate("TestSetUp", "Parameters Setup"))
        item = self.SignalTabel.horizontalHeaderItem(0)
        item.setText(_translate("TestSetUp", "Frequenz"))
        item = self.SignalTabel.horizontalHeaderItem(1)
        item.setText(_translate("TestSetUp", "Gerätenamen"))
        item = self.SignalTabel.horizontalHeaderItem(2)
        item.setText(_translate("TestSetUp", "Schalter"))
        item = self.SignalTabel.horizontalHeaderItem(3)
        item.setText(_translate("TestSetUp", "Hilfs"))
        item = self.SignalTabel.horizontalHeaderItem(4)
        item.setText(_translate("TestSetUp", "Benutzermeldung"))
        self.NewButton.setText(_translate("TestSetUp", "Neu"))
        self.CopyButton.setText(_translate("TestSetUp", "Kopieren"))
        self.EditButton.setText(_translate("TestSetUp", "Bearbeiten"))
        self.DeleteButton.setText(_translate("TestSetUp", "Löschen"))
        self.ImportButton.setText(_translate("TestSetUp", "Importieren"))
        self.systemTestButton.setText(_translate("TestSetUp", "System Testen"))
        self.ExpandButton.setText(_translate("TestSetUp", "+ Aller erweitern"))
        self.CollposeButton.setText(_translate("TestSetUp", "- Alle zusammenbrechen"))
        self.SystemControllerLabel.setText(_translate("TestSetUp", "Aktive Systemkontroller:"))
        self.KontrollerComBoBox.setToolTip(
            _translate("TestSetUp", "Wählen Sie alle Systemkontroller aus, die diesem Testaufbau zugeordnet sind."))
        self.KontrollerComBoBox.setCurrentText(_translate("TestSetUp", "None"))
        self.TestSetUpGroup.setTabText(self.TestSetUpGroup.indexOf(self.TestSetUpSignalRouting),
                                       _translate("TestSetUp", "Signal Routing"))
        self.WarningLabel.setText(_translate("TestSetUp", "Das Signal Routing ist leer!"))
        self.ApplyButton.setText(_translate("TestSetUp", "Anwenden"))
        self.CancelButton.setText(_translate("TestSetUp", "Zurück"))


if __name__ == "__main__":
    app = QtWidgets.QApplication(sys.argv)
    ui = Ui_CalibrationSetup_FS()
    ui.setupUi()
    sys.exit(app.exec_())
